api/src/main/java/org/apache/iceberg/actions/RewriteDataFiles.java

```diff
@@ -19,92 +19,120 @@
 
 package org.apache.iceberg.actions;
 
-import org.apache.iceberg.DataFile;
+import java.util.Map;
+import org.apache.iceberg.StructLike;
 import org.apache.iceberg.expressions.Expression;
 
 /**
- * An action that rewrites data files.
+ * An action for rewriting datafiles according to a Rewrite Strategy. Generally used for
```

Comment:
nit: `datafiles` -> `data files`, `Rewrite Strategy` -> `rewrite strategy`

Reply:
Corrected, RewriteStrategy is the class name though so I switched to a link. Only issue is as long as RewriteStrategy is package private the link just shows up as RewriteStrategy. This is actually an issue with us making that interface package private since an implementation in spark would have to be in a different package than the Spark action implementations which are all in `o.a.iceberg.spark.actions` and not `o.a.iceberg.actions` ... may have to think about this later